src/components/Popover/__tests__/PopoverWrapper.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  expect,
+  test,
+} from '@playwright/experimental-ct-react';
+import { PopoverWrapperForTest } from './PopoverWrapper.story';
+
+test.describe('PopoverWrapper', () => {
+  test.describe('non-visual', () => {
+    const tags = [
```

Comment:
It does not make much sense to test such big set of elements. I would test just two - div (defalut one) + e.g. `span` (or whatever you want).

But I would split it into two tests:

* default (div)
* specific tag (e.g. span)

Test for default is missing here.